lib/constants/test-statuses.ts

```diff
@@ -18,7 +18,9 @@ export enum TestStatus {
     /**
      * @note used in new UI only for rendering icons
      */
-    RETRY = 'retry'
+    RETRY = 'retry',
+    /** @note used to display tests that have both failed screenshots and errors */
+    FAIL_ERROR = 'fail_error',
```

Comment:
This is for new ui only, right?

Reply:
No, it's used in AttemptPicker, which is used both in new UI and old UI.

Reply:
If you want to put this status here, let's also mention "it could only be used in AttemptPicker", because all the other test results are really could be seen in testResult.status